src/views/router_views/psc_type/psc_type.njk

```diff
@@ -0,0 +1,54 @@
+{% extends "layouts/default.njk" %}
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+{% block main_content %}
+
+{% set pscTypeTitle = i18n.psc_type_title %}
+{% set pscTypeIndividual = i18n.psc_type_individual %}
+{% set pscTypeRle = i18n.psc_type_rle %}
+{% set pscTypeHint = i18n.psc_type_hint | safe %}
+{% set detailsTitleText = i18n.psc_type_details_title %}
+{% set detailsText = i18n.psc_type_details_text %}
+{% set continueText = i18n.global_confirm_continue_button %}
+
+<p class="govuk-body govuk-!-margin-bottom-7"></p>
+
+{{ govukRadios({
+  name: "pscType",
+  fieldset: {
+    legend: {
+      text: pscTypeTitle,
+      isPageHeading: true,
+      classes: "govuk-fieldset__legend--l"
+    }
+  },
+  items: [
+    {
+      value: "individual",
+      text: pscTypeIndividual
+    },
+    {
+      value: "rle",
+      text: pscTypeRle,
+       hint: {
+        text: pscTypeHint
+      }
+    }
+  ]
+}) }}
+
+{{ govukDetails({
+summaryText: detailsTitleText,
+text: detailsText
+}) }}
+
+{{ govukButton({
+  text: continueText,
+  href: Urls.SKELETON_THREE,
+  classes: "govuk-button--start govuk-!-margin-top-2 govuk-!-margin-bottom-8"
```

Comment:
You can remove the classes from here to get the button to display like the prototype.

Reply:
👍